.github/workflows/test.yaml

```diff
@@ -414,6 +414,9 @@ jobs:
   miri:
     name: Miri
     runs-on: ubuntu-latest
+    env:
+      # allows platform-verifier initialisation to inspect filesystem
+      MIRIFLAGS: -Zmiri-disable-isolation
```

Comment:
Huh, TIL this exists. I think there are a handful of pre-existing tests annotated to skip running under miri because they do simple file I/O. Separate from this PR maybe it's worth revisiting those with this flag enabled?

Reply:
Did a pass and reenabled as many as possible. Unfortunately that was only one, because miri is pretty allergic to ffi/crypto and most other tests involve that.